lib/generators/logidze/install/install_generator.rb

```diff
@@ -79,6 +79,18 @@ def all_functions
         def function_definitions
           @function_definitions ||= Logidze::Utils::FunctionDefinitions.from_fs
         end
+
+        # Generate `logidze_logger_after.sql` from the regular `logidze_logger.sql`
+        # by find-and-replacing a few lines
+        def generate_logidze_logger_after
+          source = File.read(File.join(__dir__, "functions", "logidze_logger.sql"))
+          source.sub!(/^CREATE OR REPLACE FUNCTION logidze_logger.*$/, "")
+          source.sub!(/^  -- version.*$/, "")
+          source.sub!("  BEGIN", "BEGIN\n    IF pg_trigger_depth() > 1 THEN\n      RETURN NULL;\n    END IF;")
```

Comment:
Is there a functional difference between `pg_trigger_depth() > 1` and `WHEN (coalesce(current_setting('logidze.disabled', true), '') <> 'on' AND pg_trigger_depth() < 1)
` in `logidze_after.sql`?

Reply:
I don't think there is. Let's move to the trigger.